lib/Context/Process.php

```diff
@@ -80,9 +80,13 @@ public function __construct($script, string $cwd = null, array $env = [], string
             "log_errors" => "1",
         ];
 
+        $otherOpts = [];
         if ($binary === null) {
             if (\PHP_SAPI === "cli") {
                 $binary = \PHP_BINARY;
+            } else if (\PHP_SAPI === "phpdbg") {
+                $binary = \PHP_BINARY;
+                $otherOpts []= '-qrr';
```

Comment:
I'm not sure whether it makes sense to start the sub-processes with `phpdbg`. Shouldn't this just fall back to the located CLI binary?

Reply:
It is often useful, for example if you need to gather coverage info without installing specific extensions (which, by the way, are way more resource-heavy than phpdbg's coverage driver).

Reply:
Generally, the coverage collection won't be enabled in the sub-process, no?

Reply:
But with this MR, you __can__ enable it if needed, for example if using directly https://github.com/sebastianbergmann/php-code-coverage, like in my case.